src/lib-components/cc-global-header.vue

```diff
@@ -0,0 +1,159 @@
+                            <a href="#" class="navbar-item navbar-link">Item 2-3</a>
+                            <a href="#" class="navbar-item navbar-link">Item 2-4</a>
+                        </div>
+                    </div>
+
+                    <div class="navbar-item has-dropdown is-hoverable" ref="outerDiv" @mouseenter="openMenu" @mouseleave="closeMenu">
+                        <button class="navbar-link is-arrowless" aria-haspopup="menu" aria-expanded="false" ref="opener" @click="toggleMenu">
+                            What we do
+                            <!--  <span class="icon"><CaretDown /></span>-->
+                            <i class="icon caret-down" />
+                        </button>
+                        <div class="navbar-dropdown">
+                            <a href="#" class="navbar-item navbar-link">Item 3-1</a>
+                            <a href="#" class="navbar-item navbar-link">Item 3-2</a>
+                        </div>
+                    </div>
+
+                    <div class="navbar-item has-dropdown is-hoverable" ref="outerDiv" @mouseenter="openMenu" @mouseleave="closeMenu">
+                        <button class="navbar-link is-arrowless" aria-haspopup="menu" aria-expanded="false" ref="opener" @click="toggleMenu">
+                            Licenses and tools
+                            <!--  <span class="icon"><CaretDown /></span>-->
+                            <i class="icon caret-down" />
+                        </button>
+                        <div class="navbar-dropdown">
+                            <a href="#" class="navbar-item navbar-link">Item 3-1</a>
+                            <a href="#" class="navbar-item navbar-link">Item 3-2</a>
+                        </div>
+                    </div>
+                     
+                    <!--Single Menu items-->
+                    <a class="navbar-item navbar-link is-arrowless" tabindex="0" href="#">
+                        News
+                        <!--  <span class="icon"><ExternalLink /></span>-->
+                        <i class="icon external-link" v-if="isExternal" />
+                    </a>
+
+                     <!--Single Menu items-->
+                    <a class="navbar-item navbar-link is-arrowless" tabindex="0" href="#">
+                        Support us
+                        <!--  <span class="icon"><ExternalLink /></span>-->
+                        <i class="icon external-link" v-if="isExternal" />
+                    </a>
+                </div>
+            </div>
+        </nav>
+  </header>
+</template>
+<script>
+    import { defineComponent } from 'vue';
+    export default defineComponent({
+        name: 'cc-golbal-header',
+        props: {
+            ariaPrimaryLabel: {
+            type: String,
+            default: "Main",
+            },
+            ariaMenuLabel: {
+            type: String,
+            default: "Menu",
+            },
+        },
```

Comment:
Indentation issues like this can be fixed automatically by a tool like `prettier`. E.g. there may already be an `npm lint` command or something similar created by the Vue CLI.